examples/ensemble/plot_hgbt_regression.py

```diff
@@ -0,0 +1,449 @@
+"""
+=============================================================
+Usecase of advanced features in Histogram Boosting Regression
+=============================================================
+
+:ref:`histogram_based_gradient_boosting` (HGBT) models may be the most useful
+supervised learning models in scikit-learn. They are based on a modern gradient
+boosting implementation comparable to LightGBM and XGBoost. As such, HGBT models
+are more feature rich than -and often outperforms- alternative models like
+random forests, especially when the number of samples is larger than tens of
+thousands of samples (see
+:ref:`sphx_glr_auto_examples_ensemble_plot_forest_hist_grad_boosting_comparison.py`).
+
+The top usability features of HGBT models are:
+
+- :ref:`categorical_support_gbdt` (see
+  :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_categorical.py`).
+- :ref:`nan_support_hgbt`, which avoids the need for an imputer.
+- :ref:`Quantile loss support <quantile_support_hgbdt>`.
+- :ref:`monotonic_cst_gbdt`.
+- :ref:`_interaction_cst_hgbt`.
+- early stopping
+
+Note that random forests have none of those capabilities.
```

Comment:
Ss mentioned, RF has monotonicity constraits with v1.4.

Reply:
Addressed in [86f8f67](https://github.com/scikit-learn/scikit-learn/pull/26991/commits/86f8f6785161a6ceea99889d3fd40eb85648f5e5).